buildconfig/Jenkins/Conda/nightly_build_and_deploy.jenkinsfile

```diff
@@ -1,16 +1,16 @@
 def git_clone(){
     git url: 'https://github.com/mantidproject/mantid.git',
-        branch: 'main_pipeline_prototype'
```

Comment:
Silly question: if Jenkins has already cloned the repo to grab this file why to we need to clone it again?

Reply:
Jenkins I am fairly certain just grabs a copy of the file that is the .jenkinsfile, as it manages to complete this far to quickly to be a full clone. Unless @DavidFair knows different?

Reply:
I have found the magic in this article: https://ikeptwalking.com/how-to-checkout-code-in-jenkins-pipeline/

We just want to write:
```
scm checkout
```
and it will use the job configuration directly.

Reply:
Unfortunately we tried that yesterday, I just ran it again, so here is the output from that:
https://builds.mantidproject.org/job/main_nightly_deployment_prototype/179/console

Reply:
As agreed we'll hardcode this as `release-next`